pkg/storage/plugins/mongodb/mongodb.go

```diff
@@ -201,6 +201,7 @@ func (s *Store) Find(ctx context.Context, opts plugins.FindOptions) ([]bson.Raw,
 	if err != nil {
 		return nil, span.Error(err)
 	}
+	defer cur.Close(ctx)
```

Comment:
I wonder if there's an IDE plugin that will catch when I forgot to call close on variables that I create? This needs to exist. 😁

Reply:
maybe someone wrote a linter for it but I can see how it can be hard to detect it

Reply:
I found two linters for specific libraries that check if Close was called, but not a general purpose "did I close things that I created" check

https://github.com/golangci/awesome-go-linters